docs/thermometer-skills.md

```diff
@@ -0,0 +1,267 @@
+The AEA thermometer skills demonstrate an interaction between two AEAs.
+
+* The provider of thermometer data (the thermometer).
+* The buyer of thermometer data (the thermometer client).
+
+## Preparation instructions
+ 
+### Dependencies
+
+Follow the <a href="../quickstart/#preliminaries">Preliminaries</a> and <a href="../quickstart/#installation">Installation</a> sections from the AEA quick start.
+
+##Discussion
+
+The scope of the specific demo is to demonstrate how to create a very simple AEA with the usage of the AEA framework, a Raspberry Pi, and a thermometer sensor. The thermometer AEA
+will read data from the sensor each time a client requests and will deliver to the client upon payment. To keep the demo simple we avoided the usage of a database since this would increase the complexity. As a result, the AEA can provide only one reading from the sensor.
+Another step that we avoided is the usage of a smart contract that could store the readings from the sensor. As a result, we interact with a ledger only to complete a transaction.
+
+Since the AEA framework enables us to use third-party libraries we don't have to do something to be able to read from the sensor. The `aea install` command will install each dependency that the specific AEA needs and is listed in the skill's YAML file. 
```

Comment:
Change first sentence to: Since the AEA framework enables us to use third-party libraries hosted on PyPI we can directly reference the external dependencies.